packages/experience-builder-sdk/src/core/editor/designComponentUtils.ts

```diff
@@ -15,14 +15,18 @@ export const deserializeDesignComponentNode = ({
   node,
   nodeId,
   parentId,
-  experienceDataSource,
-  experienceUnboundValues,
+  designComponentDataSource,
+  designComponentUnboundValues,
+  componentInstanceProps,
+  componentInstanceUnboundValues,
 }: {
   node: CompositionNode;
   nodeId: string;
   parentId?: string;
-  experienceDataSource: CompositionDataSource;
-  experienceUnboundValues: CompositionUnboundValues;
+  designComponentDataSource: CompositionDataSource;
+  designComponentUnboundValues: CompositionUnboundValues;
```

Comment:
Will we ever have a need for the `designComponentDataSource` and `designComponentUnboundValues`? 

Because looking at line 41 -51, we make use of the componentInstanceUnboundValues and we'd also probably do the same when we start dealing with the data source, so we never really take a look at the unbound values object in the design component.

Except we use it as a fallback value for when there `componentInstanceUnboundValues` is empty or does not have the value

Reply:
They contain the default value so we could use it as a fallback but theoretically, we copy over everything and thus don't need to look them up here again, right?

Reply:
I think that should be the case - using it as a fallback. But there's a miscommunication somewhere between the `designComponentUnboundValues` and the variable mapping, coz the uuids are totally different (which shouldn't be). 

Look at the `unboundValues` in the instance and the design component unboundValues in this screenshot. The uuids don't match at all, so we can't do a look up there. 

<img width="1045" alt="Screenshot 2023-11-22 at 11 31 00" src="https://github.com/contentful/experience-builder/assets/30434146/bcdbbc07-a3c9-4e03-8d1e-7dcf2413ada1">

Reply:
We cannot copy same `uuid`s of `UnboundValues` from design component to its instance. In a scenario where customer is using an instance of design component multiple times in Experience.  `uuid`s should be different then.

Reply:
I think it matches right now the developer components, doesn't it? If you would change there a default values it would overwrite existing instances of those as the default values are copied over, correct?